src/components/ReportPage/index.jsx

```diff
@@ -459,8 +456,8 @@ export const mapDispatchToProps = dispatch => ({
   fetchAllAutomation: (pagination, filters) => dispatch(fetchAutomation(pagination, filters)),
   fetchUpdates: () => dispatch(fetchRealTimeReport()),
   resetUpdates: () => dispatch(resetRealTimeReport()),
-  fetchStat: period => dispatch(fetchStatsRequest(period)),
-  retryFailedAutomation: automationId => dispatch(retryAutomation(automationId)),
+  fetchStat: (period) => dispatch(fetchStatsRequest(period)),
+  retryFailedAutomation: () => dispatch(retryAutomation()),
```

Comment:
Hey, @blackshady  retryAutomations expects AutomationId but you have taken it out. Is this intended?

https://github.com/andela/bp-esa-frontend/blob/28fb2fc768824f0bd129ef380e008548153da6b1/src/redux/actions/automationActions.js#L27